md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessDAO.java

```diff
@@ -702,5 +726,97 @@ else  if (childProcess.getProcessType().getProcessTypeId() == 36){
         return processList;
     }
 
+public String securityCheck(Integer processId,String username,String action){
+    Session session = sessionFactory.openSession();
+    session.beginTransaction();
+    Process process = (Process) session.get(Process.class, processId);
+    Criteria criteria = session.createCriteria(UserRoles.class).add(Restrictions.eq("users.username", username));
+    List<UserRoles> userRoles = criteria.list();
+    List<String> userRolesNameList=new ArrayList<>();
+    String processCreater=process.getUserRoles().getRole();
+    for(UserRoles userRoles1:userRoles)
+    {userRolesNameList.add(userRoles1.getRole());}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)